src/linear_algebra/direct_sum_module.lean

```diff
@@ -6,6 +6,7 @@ Authors: Kenny Lau
 Direct sum of modules over commutative rings, indexed by a discrete type.
 -/
 import algebra.direct_sum
+import linear_algebra.dfinsupp
```

Comment:
you can probably remove the `linear_algebra.basic` import just after this, right?

Reply:
I could - does mathlib have anything to say about whether this is preferred / frowned upon?

Reply:
The tendency is to keep the list of `import`s as short as possible. If we want to know transitive dependencies, `leanproject` can generate the graph.